CSSTooltipPseudo/explainer.md

```diff
@@ -0,0 +1,896 @@
+    - [Gecko](#gecko-1)
+    - [Webkit](#webkit-1)
+- [The `::tooltip` pseudo element](#the-tooltip-pseudo-element)
+  - [How to trigger `::tooltip` default styles?](#how-to-trigger-tooltip-default-styles)
+  - [How to set the `::tooltip` content?](#how-to-set-the-tooltip-content)
+  - [Positioning for `::tooltip`](#positioning-for-tooltip)
+  - [Default styles for `::tooltip`](#default-styles-for-tooltip)
+  - [What styles can be applied to `::tooltip`?](#what-styles-can-be-applied-to-tooltip)
+  - [`::tooltip` style inheritance](#tooltip-style-inheritance)
+  - [What user interaction triggers the built-in tooltip?](#what-user-interaction-triggers-the-built-in-tooltip)
+  - [Dependencies on non-stable features](#dependencies-on-non-stable-features)
+- [Key scenarios](#key-scenarios)
+  - [Scenario 1: Opt-into the new base `::tooltip` styles](#scenario-1-opt-into-the-new-base-tooltip-styles)
+  - [Scenario 2: Adjusting the look and feel of built-in tooltips](#scenario-2-adjusting-the-look-and-feel-of-built-in-tooltips)
+  - [Scenario 3: Adjusting tooltip timing](#scenario-3-adjusting-tooltip-timing)
+  - [Scenario 4: Setting the name using `title` without rendering a tooltip](#scenario-4-setting-the-name-using-title-without-rendering-a-tooltip)
+  - [Scenario 5: Providing custom positioning to a tooltip](#scenario-5-providing-custom-positioning-to-a-tooltip)
+- [Accessibility Considerations](#accessibility-considerations)
+  - [What is the current accessibility experience of tooltips across browsers?](#what-is-the-current-accessibility-experience-of-tooltips-across-browsers)
+  - [Proposed improvements](#proposed-improvements)
+  - [Zoom](#zoom)
+- [Security Considerations](#security-considerations)
+- [Privacy Considerations](#privacy-considerations)
+- [Open questions](#open-questions)
+- [Future ideas](#future-ideas)
+  - [Tootlip pointer/arrow](#tootlip-pointerarrow)
+  - [Customizing `::tooltip` user interactions](#customizing-tooltip-user-interactions)
+  - [CSS Anchor Positioning Tooltip Defaults](#css-anchor-positioning-tooltip-defaults)
+- [Considered alternatives](#considered-alternatives)
+  - [Alternative 1: The `interesttarget` attribute](#alternative-1-the-interesttarget-attribute)
+- [References & acknowledgements](#references--acknowledgements)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction
+
+Tooltips are used by authors to provide additional contextul information to users
+for a given interactive element on the screen. Tooltips are often presented as
+a textbox that appears on hover (or potentially some other user interaction, like
+keyboard focus, or long-press on a touchscreen device) on the element the
+tooltip is associated with.
+
+Authors can utilize the browser's built-in tooltips by setting the
+[`title`](https://html.spec.whatwg.org/multipage/dom.html#attr-title)
+attribute on an element of interest. However, the tooltip that appears as a
```

Comment:
Might also be worth calling out the SVG `<title>` element, which works similarly: https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Element/title

Reply:
Good call out. Forgot to add it at this location. Updated!